tests/test_axis.py

```diff
@@ -1,52 +1,17 @@
 import numpy as np
```

Comment:
The variables in `setup()` were moved to `tests/fixtures.py`.
Review can be skipped since tests still pass.